reana_commons/validation/schemas/reana_analysis_schema.json

```diff
@@ -89,7 +89,10 @@
         },
         "resources": {
           "$id": "/properties/workflow/properties/resources",
-          "type": "object",
+          "type": [
+            "object",
+            "null"
```

Comment:
In which cases `resources` can be null?

Reply:
Never - restored it to just allowing objects